packages/block-library/src/post-tags/index.js

```diff
@@ -14,5 +15,7 @@ export { metadata, name };
 
 export const settings = {
 	title: __( 'Post Tags' ),
+	description: __( 'Insert a list of tags to your document.' ),
```

Comment:
I don't think this adequately describes what the block does.

It displays the list of tags for the current document.